README.md

```diff
@@ -1,39 +1,30 @@
-# PG2: A *Fast* PostgreSQL Driver For Clojure
+# PG2: *Fast* PostgreSQL driver for Clojure
 
-[pg]: https://github.com/igrishaev/pg
+`PG2` is a JDBC-free PostgreSQL client library.
 
-PG2 is a client library for PostgreSQL server. It succeeds [PG(one)][pg] -- my
```

Comment:
sorry I removed it since it seems to be a bit off-topic, IMO it's important on https://github.com/igrishaev/pg only

or maybe it can be moved to the bottom as a history, it's your call ofc